busy_beaver/adapters/meetup.py

```diff
@@ -44,16 +46,24 @@ def create_event_record(self) -> Event:
 class MeetupAdapter:
     """Pull the upcoming events from Meetup and send the message to Slack."""
 
-    def __init__(self, api_key):
-        self.meetup_client = MeetupClient(api_key)
+    def __init__(self, oauth_token: str):
+        default_headers = {"Authorization": f"Bearer {oauth_token}"}
+        self.client = RequestsClient(headers=default_headers)
 
     def get_events(self, group_name: str, count: int = 1) -> List[EventDetails]:
-        events = self.meetup_client.GetEvents(group_urlname=group_name)
-        if not events.results:
+        url = BASE_URL + f"/{group_name}/events"
```

Comment:
Only hitting one endpoint so no worries that this function is larger than normal